src/VirtoCommerce.ExperienceApiModule.DigitalCatalog/Mapping/CategoryMappingProfile.cs

```diff
@@ -1,14 +1,26 @@
 using AutoMapper;
 using VirtoCommerce.ExperienceApiModule.Core.Binding;
 using VirtoCommerce.SearchModule.Core.Model;
+using VirtoCommerce.StoreModule.Core.Model;
 
 namespace VirtoCommerce.XDigitalCatalog.Mapping
 {
     public class CategoryMappingProfile : Profile
     {
         public CategoryMappingProfile()
         {
-            CreateMap<SearchDocument, ExpCategory>().ConvertUsing(src => new GenericModelBinder<ExpCategory>().BindModel(src) as ExpCategory);
+            CreateMap<SearchDocument, ExpCategory>().ConvertUsing((src, dest, context) =>
+            {
+                var expCategory = new GenericModelBinder<ExpCategory>().BindModel(src) as ExpCategory;
+
+                if (expCategory != null)
+                {
+                    expCategory.Store = context.Options.Items["store"] as Store;
+                    expCategory.CultureName = context.Options.Items["language"].ToString();
```

Comment:
Change `language` to `cultureName`, please.

Reply:
done .